data-pipeline-flink/user-cache-updater-2.0/src/main/scala/org/sunbird/dp/usercache/functions/UserCacheUpdaterFunctionV2.scala

```diff
@@ -20,25 +20,24 @@ class UserCacheUpdaterFunctionV2(config: UserCacheUpdaterConfigV2)(implicit val
 
   private[this] val logger = LoggerFactory.getLogger(classOf[UserCacheUpdaterFunctionV2])
   private var dataCache: DataCache = _
-  private var cassandraConnect: CassandraUtil = _
-  private var custodianOrgId: String = _
+
+  private var restUtil: RestUtil = _
```

Comment:
Does the RestUtil do an automatic retry for n number of times before failing?

Reply:
The Apache HTTPClient retries automatically retries for 3 times.